src/Microsoft.AspNetCore.Mvc.Core/DependencyInjection/IMvcBuilder.cs

```diff
@@ -3,8 +3,14 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
+    /// <summary>
+    /// Marker interface for configuring MVC services.
```

Comment:
What is "marker" about this interface? Usually reserve that word for empty interfaces that just say "treat me special".